src/views/spotlights/housing/index.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <a ref="scrollRef" class="scroll-anchor" />
+  <Dashboard :dataset-name="datasetName" />
+</template>
+
+<script setup lang="ts">
+// import { ref } from "vue";
```

Comment:
Lots of commented out code in this file. Let's take it out? Is there a reason it's still here?